charts/cloudstack-csi/Chart.yaml

```diff
@@ -2,10 +2,13 @@ apiVersion: v2
 name: cloudstack-csi
 description: A Helm chart for CloudStack CSI driver
 type: application
-version: 2.0.1
-appVersion: 0.6.1
```

Comment:
Why is appVersion removed here? It will break the chart.

Reply:
It was by mistake fixed with the last commit.